src/reactA11yIframesRule.ts

```diff
@@ -0,0 +1,99 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+import * as tsUtils from 'tsutils';
+
+import { ExtendedMetadata } from './utils/ExtendedMetadata';
+import { getJsxAttributesFromJsxElement } from './utils/JsxAttribute';
+
+const IFRAME_ELEMENT_NAME: string = 'iframe';
+const TITLE_ATTRIBUTE_NAME: string = 'title';
+const SRC_ATTRIBUTE_NAME: string = 'src';
+const HIDDEN_ATTRIBUTE_NAME: string = 'hidden';
+const IFRAME_EMPTY_TITLE_ERROR_STRING: string = 'An iframe element must have a non-empty title.';
+const IFRAME_EMPTY_OR_HIDDEN_ERROR_STRING: string = 'An iframe element should not be hidden or empty.';
+const IFRAME_UNIQUE_TITLE_ERROR_STRING: string = 'An iframe element must have a unique title.';
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static metadata: ExtendedMetadata = {
+        ruleName: 'react-a11y-iframes',
+        type: 'functionality',
+        description: 'Enforce that iframe elements are not empty, have title and are unique.',
```

Comment:
> have title and are

Nit: a [serial comma](https://en.wikipedia.org/wiki/Serial_comma) should be after 'title'. Here and in the README.md please.

Reply:
on it :books:

Reply:
Looks like this one is not changed.
@JoshuaKGoldberg do I understand correctly that `,` is required after `title` here?